README.adoc

```diff
@@ -25,13 +25,15 @@ endif::[]
 
 == Description
 
-You can use the `transformheaders` policy is to override HTTP headers in incoming requests or outbound responses.
+You can use the `transformheaders` policy to override HTTP headers in incoming requests or outbound responses.
 You can override the HTTP headers by:
 
 * Adding to or updating the list of headers
 * Removing headers individually
 * Defining a whitelist
 
+Please note that on the request-content phease, you can only add new headers.
```

Comment:
IMHO I don't think there is any constraints regarding the headers during the request_content or response_content phases.

Headers are sent at the end of these phases so I don't see any reason why it could be impossible to update or remove an header (and why adding a new header would work).

Can you have a try to make sure it is really impossible because it is really intriguing ? Thx